lib/new_relic/util.ex

```diff
@@ -77,7 +78,9 @@ defmodule NewRelic.Util do
 
   @aws_vendor_data ["availabilityZone", "instanceId", "instanceType"]
   defp aws_vendor_hash(url) do
-    case HTTPoison.get(url, [], timeout: 100) do
+    opts = NewRelic.Config.httpoison_opts()
+
+    case HTTPoison.get(url, [], [timeout: 100] ++ opts) do
```

Comment:
This is making an attempt to call a port on localhost that only exists in AWS, so we wouldn't apply any proxy settings here